native/acter/src/api/pins.rs

```diff
@@ -370,11 +370,23 @@ impl PinDraft {
 
     pub async fn send(&self) -> Result<OwnedEventId> {
         let room = self.room.clone();
+        let my_id = room
+            .client()
+            .user_id()
+            .context("User not found")?
+            .to_owned();
         let content = self.content.build()?;
+
         RUNTIME
             .spawn(async move {
-                let resp = room.send(content).await?;
-                Ok(resp.event_id)
+                let permitted = room
+                    .can_user_send_message(&my_id, MessageLikeEventType::RoomMessage)
+                    .await?;
```

Comment:
hmm.... not super sure about this one ... the goal is to have type-specific power-levels for our own objects, too ... this implies that anyone can send them ... while this is fine for now, it isn't actually accurate...

Reply:
For now, we have no ways to support power levels separately for pin, news or attachment.
This issue will be fixed in another PR.